bundles/org.openhab.binding.solax/README.md

```diff
@@ -0,0 +1,210 @@
+| pv2Power        | system.electric-power      | The output power PV2 string [W]                 |
+| pvTotalPower    | system.electric-power      | The total output power of both PV strings [W]   |
+| pvTotalCurrent  | system.electric-current    | The total current of both PV strings [A]        |
+
+### Battery channels
+
+| Channel                  | Type                       | Description                                                                                    |
+|--------------------------|----------------------------|------------------------------------------------------------------------------------------------|
+| batteryPower             | system.electric-power      | The power to / from battery (negative means power is pulled from battery and vice-versa) [W]   |
+| batteryCurrent           | system.electric-current    | The current to / from battery (negative means power is pulled from battery and vice-versa) [A] |
+| batteryVoltage           | system.electric-voltage    | The voltage of the battery [V]                                                                 |
+| batteryTemperature       | Number:Temperature         | The temperature of the battery [C/F]                                                           |
+| batteryStateOfCharge     | system.battery-level       | The state of charge of the battery [%]                                                         |
+
+### Grid related channels
+
+| Channel                  | Type                       | Description                                                                                    |
+|--------------------------|----------------------------|------------------------------------------------------------------------------------------------|
+| feedInPower              | system.electric-power      | The power to / from grid (negative means power is pulled from the grid and vice-versa) [W]     |
+| onGridTotalYield*        | system.electric-power      | Total Yield from PV strings [W]     |
+| onGridDailyYield*         | system.electric-power      | The power to / from grid (negative means power is pulled from the grid and vice-versa) [W]     |
+| onTotalFeedinEnergy*      | system.electric-power      | The power to / from grid (negative means power is pulled from the grid and vice-versa) [W]     |
+| onTotalFeedinEnergy*      | system.electric-power      | The power to / from grid (negative means power is pulled from the grid and vice-versa) [W]     |
+
+_* experimental channels. So far the data does not seem reliable. Either it's taken from wrong part of the JSON or it's calculated wrong in the inverter itself. If someone finds better way to parse it or the meaning of the various int values, please open a new issue for solax binding and share the information._
+
+### General channels
+
+| Channel                  | Type                       | Description                                                                                    |
```

Comment:
Can the formatting of this table (in source form) be fixed?

Reply:
Checked and fixed them all to have proper identation of the vertical lines